apps/src/gamelab/ErrorDialogStack.jsx

```diff
@@ -3,39 +3,117 @@ import React, {PropTypes} from 'react';
 import * as actions from './errorDialogStackModule';
 import {connect} from 'react-redux';
 import BaseDialog from '../templates/BaseDialog.jsx';
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+import gamelabMsg from '@cdo/gamelab/locale';
+import msg from '@cdo/locale';
+import Button from '@cdo/apps/templates/Button';
+import DialogFooter from '@cdo/apps/templates/teacherDashboard/DialogFooter';
+import * as animationActions from './animationListModule';
+import firehoseClient from '@cdo/apps/lib/util/firehose';
 
 /**
  * Renders error dialogs in sequence, given a stack of errors.
  */
 class ErrorDialogStack extends React.Component {
   static propTypes = {
+    //From redux
     errors: PropTypes.arrayOf(PropTypes.object).isRequired,
-    dismissError: PropTypes.func.isRequired
+    dismissError: PropTypes.func.isRequired,
+    deleteAnimation: PropTypes.func,
+    animationList: PropTypes.object
   };
 
+  handleDeleteChoice(key){
+    // Log data about when this scenario occurs
+    firehoseClient.putRecord(
+     'analysis-events',
```

Comment:
indent +2 from previous line (+1 from what you have here), on this line and in other blocks below